### PR TITLE
Update golem-certificate version

### PR DESCRIPTION
**Pull request by evik42**
Resolves: https://github.com/golemfactory/golem-certificate/issues/19

Update golem-certificate version to update to finalized schema IDs.

### CODE REVIEW

**Review by MrDarthShoe — DISMISSED** [review]

File: goth_tests/e2e/vm/assets/test_e2e_rule_partner_outbound/provider/cert-dir/cert-permissions.json
Comment (MrDarthShoe) [inline goth_tests/e2e/vm/assets/test_e2e_rule_partner_outbound/provider/cert-dir/cert-permissions.json:1]: This should be removed I believe

File: goth_tests/e2e/vm/assets/test_e2e_rule_partner_outbound/provider/rules.json
Comment (MrDarthShoe) [inline goth_tests/e2e/vm/assets/test_e2e_rule_partner_outbound/provider/rules.json:1]: Why `80c84b2701126669966f46c1159cae89c58fb088e8bf94b318358fa4ca33ee56d8948511a397e5aba6aa5b88fff36f2541a91b133cde0fb816e8592b695c04c3` and `cb16a2ed213c1cf7e14faa7cf05743bc145b8555ec2eedb6b12ba0d31d17846d2ed4341b048f2e43b1ca5195a347bfeb0cd663c9e6002a4adb7cc7385112d3cc` are in both rules? I believe they should be set only for one.

File: goth_tests/e2e/vm/test_e2e_rule_partner_outbound.py [hunk 1]
```diff
@@ -41,7 +41,26 @@ async def test_e2e_rule_partner_outbound(
         {"name": "requestor", "type": "Requestor", "address": "d1d84f0e28d6fedf03c73151f98df95139700aa7" },
         {"name": "provider-1", "type": "VM-Wasm-Provider", "address": "63fc2ad3d021a4d7e64323529a55a9442c444da0", "use-proxy": True},
     ]
+
+    assets_root = Path(__file__).parent / "assets";
+    node_types = [
```
Comment (MrDarthShoe) [inline goth_tests/e2e/vm/test_e2e_rule_partner_outbound.py:46]: Could you please add comments about why it is needed? Here and in other e2e test so "knowledge" will stay with us.

File: goth_tests/e2e/vm/assets/test_e2e_rule_partner_outbound/provider/cert-dir/cert-permissions.json
Comment (evik42) [reply goth_tests/e2e/vm/assets/test_e2e_rule_partner_outbound/provider/cert-dir/cert-permissions.json:1]: it is present in the default assets

File: goth_tests/e2e/vm/assets/test_e2e_rule_partner_outbound/provider/rules.json
Comment (evik42) [reply goth_tests/e2e/vm/assets/test_e2e_rule_partner_outbound/provider/rules.json:1]: this is a copy from default assets

File: goth_tests/e2e/vm/test_e2e_rule_partner_outbound.py (hunk 1 above)
Comment (evik42) [reply goth_tests/e2e/vm/test_e2e_rule_partner_outbound.py:46]: this is temporary, will remove it once the new goth default assets are published

File: goth_tests/e2e/vm/assets/test_e2e_rule_partner_outbound/provider/cert-dir/cert-permissions.json
Comment (MrDarthShoe) [reply goth_tests/e2e/vm/assets/test_e2e_rule_partner_outbound/provider/cert-dir/cert-permissions.json:1]: Created task for it https://github.com/golemfactory/yagna/issues/2634

File: goth_tests/e2e/vm/assets/test_e2e_rule_partner_outbound/provider/rules.json
Comment (MrDarthShoe) [reply goth_tests/e2e/vm/assets/test_e2e_rule_partner_outbound/provider/rules.json:1]: created task for it
https://github.com/golemfactory/yagna/issues/2634

**Review by MrDarthShoe — APPROVED** [review]

**Pull request merged**